tests/functional/base_test.go

```diff
@@ -132,6 +132,54 @@ func TwoNodeBaremetalSetSpecWithWrongNodeLabel(namespace string) map[string]inte
 	return spec
 }
 
+func MultiNodeBaremetalSetSpecWithSameNodeLabel(namespace string) map[string]interface{} {
+	spec := map[string]interface{}{
+		"baremetalHosts": map[string]interface{}{
+			"compute-0": map[string]interface{}{
+				"ctlPlaneIP":       "10.0.0.1",
+				"bmhLabelSelector": map[string]string{"nodeType": "compute"},
+			},
+			"compute-1": map[string]interface{}{
+				"ctlPlaneIP":       "10.0.0.1",
```

Comment:
Should we use different IPs between the hosts to have more parity with real CRs?

Reply:
done! they're not relevant in the webhook tests though.

Reply:
Fair point in that regard, but thanks for updating nonetheless :+1: